src/browser/modules/Sidebar/Settings.jsx

```diff
@@ -228,12 +234,13 @@ export const Settings = ({
                 checked={settings[setting]}
                 data-testid={setting}
               />
-              <StyledSettingLabel title={tooltip}>{visual}</StyledSettingLabel>
-            </StyledSetting>
-          )
-        }
-      })
-      .filter(setting => setting !== false)
```

Comment:
I removed this fail-safe check since I figure it's better to have the sidebar crash on malformed form items. Thoughts?